target/snitch_cluster/Makefile

```diff
@@ -148,6 +148,25 @@ $(eval $(call generate_snax_gen,snax_streamer_gemmX))
  
 endif
 
+ifeq (${CFG_OVERRIDE}, cfg/snax-streamer-gemmX-data-reshuffler.hjson)
+ 
+$(eval $(call generate_snax_gen,snax_streamer_gemmX))
+
+	SNAX_GEMMX_ROOT ?= $(ROOT)/hw/chisel_acc
+    include $(SNAX_GEMMX_ROOT)/Makefile
+ 
+    VSIM_BENDER += -t snax_streamer_gemmX
+    VLT_BENDER  += -t snax_streamer_gemmX
+    VCS_BENDER  += -t snax_streamer_gemmX
+
+$(eval $(call generate_snax_gen,snax_data_reshuffler))
+
+	VSIM_BENDER += -t snax_data_reshuffler
+	VLT_BENDER  += -t snax_data_reshuffler
+	VCS_BENDER  += -t snax_data_reshuffler
+
+endif
```

Comment:
```suggestion
ifeq (${CFG_OVERRIDE}, cfg/snax-streamer-gemmX-data-reshuffler.hjson)
 
$(eval $(call generate_snax_gen,snax_streamer_gemmX))

    SNAX_GEMMX_ROOT ?= $(ROOT)/hw/chisel_acc
    include $(SNAX_GEMMX_ROOT)/Makefile
 
    VSIM_BENDER += -t snax_streamer_gemmX
    VLT_BENDER  += -t snax_streamer_gemmX
    VCS_BENDER  += -t snax_streamer_gemmX

$(eval $(call generate_snax_gen,snax_data_reshuffler))

    VSIM_BENDER += -t snax_data_reshuffler
    VLT_BENDER  += -t snax_data_reshuffler
    VCS_BENDER  += -t snax_data_reshuffler

endif
```

Reply:
Maybe #112 can avoid all of this code 😸

Reply:
Hi @jorendumoulin , we need two targets here, not merely the filename. Do you have any thoughts about this?